checks/dockerhost.go

```diff
@@ -134,9 +134,13 @@ func CheckTrustedUsers(client *client.Client) (res Result) {
 }
 
 func AuditDockerDaemon(client *client.Client) (res Result) {
-	var ruleExists bool
 	res.Name = "1.7 Audit docker daemon"
-	ruleExists = checkAuditRule("/usr/bin/docker")
+	ruleExists, err := checkAuditRule("/usr/bin/docker")
+	if err != nil {
+		output := fmt.Sprintf("An error occured: %s", err)
+		res.Skip(output)
+		return
+	}
 
 	if ruleExists {
```

Comment:
This is not necessary, since if the `checkAuditRule("/usr/bin/docker")` returns an error, it returns early.